### DISCUSSION TITLE
query_engine issue

### DISCUSSION THREAD

**Discussion by yangboz** [safevideo/autollm#231] [Q&A] [answered]
code example comes from https://abrahimzaman360.medium.com/introduction-to-autollm-c8cd31be2a5f

after run it ,throws error as following:
```
query_engine = AutoQueryEngine.from_parameters(documents=documents)
AttributeError: type object 'AutoQueryEngine' has no attribute 'from_parameters'
```
any ideas? thanks

**Comment by yangboz** [comment]
maybe more documents on query_engine?

**Comment by fcakyon** [comment]
Hello @yangboz. Please refer to readme for the most up-to-date usage: https://github.com/safevideo/autollm?tab=readme-ov-file#create-a-query-engine-in-seconds

**Reply by yangboz** [reply]
I mean  any query_engine with local Ollama code example ? 

thanks

**Reply by fcakyon** [reply, marked as answer]
You just have to provide an ollama model name as `llm_model="ollama/llama2"` and ollama endpoint URL as `llm_api_base="http://localhost:11434"` in query_engine.

Feel free to open a PR for adding this information into readme :) @yangboz

**Reply by yangboz** [reply]
after run it , error comes as following:

```
query_engine = AutoQueryEngine.from_defaults(
    ^
SyntaxError: keyword argument repeated: llm_model
```
any idea ? thanks.

my bad , just changed the variable name to differ one.

**Reply by fcakyon** [reply]
@yangboz you might be using a deprecated autollm version, try to update to latest please.

**Reply by yangboz** [reply]
ok, I will try to use the latest version.

**Comment by yangboz** [comment]
despite of  previous error, after export OPEN_AI_KEY,  calling queryEngine.query ,coming error as following:
```
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLZeroReturnError(6, 'TLS/SSL connection has been closed (EOF) (_ssl.c:1129)'))': /api/4505950933614592/envelope/
``` 
any idea? thanks.

**Reply by fcakyon** [reply]
what is your autollm version?
